osu.Game/Screens/SelectV2/Carousel.cs

```diff
@@ -678,6 +679,15 @@ private static void expirePanelImmediately(Drawable panel)
             carouselPanel.Expanded.Value = false;
         }
 
+        protected override bool OnInvalidate(Invalidation invalidation, InvalidationSource source)
```

Comment:
Is there a reason you've used this instead of a `LayoutValue`? Primary concern is you're not returning `true` here with means the drawnode MAY not get invalidated if the stars line up correctly and the base call doesn't do it for you, which is an easy thing to miss by using this method.

Reply:
hmm, it's copy-pasted from old carousel. can try using the newer structure.

Reply:
It's a non blocker anyway, was more interested than anything.